book/src/appendices/a_config_files/arena_config.md

```diff
@@ -0,0 +1,120 @@
+# Adding an Arena Config
+
+To begin with, let's make the `Arena` dimensions configurable. Add this structure to a new file `config.rs`.
+```rust,ignore
+#[derive(Debug, Deserialize, Serialize)]
+struct ArenaConfig {
+    pub height: f32,
```

Comment:
This might be a good time to introduce [`[#serde(default)]`](https://serde.rs/attr-default.html)